omod/src/main/resources/webModuleApplicationContext.xml

```diff
@@ -23,7 +23,6 @@
 	<!-- Beans related to the web context -->
 	 
 	<!-- Annotation based controllers -->

```

Comment:
Get rid of the comment as well, otherwise we end up with a stale comment

Reply:
ok @ibacher  let me do it now